include/mc_tasks/MetaTask.h

```diff
@@ -49,13 +49,13 @@ struct MC_SOLVER_DLLAPI MetaTask
    * The name should be set before being added to the solver.
    *
    */
-  void name(const std::string & name)
+  virtual inline void name(const std::string & name)
   {
     name_ = name;
   }
 
   /** Get the name of the task */
-  inline const std::string & name() const
+  virtual inline const std::string & name() const
```

Comment:
I don't think this needs to be `virtual`

Reply:
Without `virtual`, I get error with `override`.
```
/home/mmurooka/jrl/mc_rtc/include/mc_tasks/lipm_stabilizer/StabilizerTask.h:98:8: error: ‘void mc_tasks::lipm_stabilizer::StabilizerTask::name(const string&)’ marked ‘override’, but does not override
   void name(const std::string & name) override
        ^~~~
```

Reply:
That's weird, the getter and the setters are different functions.[This](https://godbolt.org/z/5zvqhsW5v) works without any issue

Reply:
Thank you for the verification code. Sorry, I made a mistake and removed `virtual` from setter method.

Fixed all the three review comments in https://github.com/jrl-umi3218/mc_rtc/pull/171/commits/2d02461b8381c61735a09254cb43c13ebb41033b